src/TomcatSeverTreeProvider.ts

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+import * as fse from "fs-extra";
+import * as path from "path";
+import * as vscode from "vscode";
+import { TreeItem } from "vscode";
+import { ServerState } from "./Constants";
+import { TomcatModel } from "./Tomcat/TomcatModel";
+import { TomcatServer } from "./Tomcat/TomcatServer";
+import { WarPackage } from "./Tomcat/WarPackage";
+import { Utility } from "./Utility";
+
+export class TomcatSeverTreeProvider implements vscode.TreeDataProvider<TreeItem> {
+    public _onDidChangeTreeData: vscode.EventEmitter<TreeItem> = new vscode.EventEmitter<TreeItem>();
+    public readonly onDidChangeTreeData: vscode.Event<TreeItem> = this._onDidChangeTreeData.event;
+
+    constructor(private _context: vscode.ExtensionContext, private _tomcatModel: TomcatModel) {
+        this._onDidChangeTreeData.fire();
+    }
+
+    public async getTreeItem(element: TreeItem): Promise<TreeItem> {
+        /* qisun: checking http port is not a valid way to detect the server running or not
```

Comment:
remove the comments if they are not used anymore

Reply:
Agreed, just delete whatever is outdated.